tests/posix/signal/prj.conf

```diff
@@ -0,0 +1,5 @@
+CONFIG_POSIX_API=y
+CONFIG_ZTEST=y
+
+CONFIG_POSIX_AEP_CHOICE_PSE51=y
+CONFIG_POSIX_SIGNALS=y
```

Comment:
As noted from the CI, an empty line is always needed at the end of the file. Installing the `editorconfig` plugin (if you are using vscode) would take care of such nitty gritty for you

Reply:
checking these out thank you.